### PR TITLE
View-course added in survey list (connects #3564)

### PR DESCRIPTION
**Pull request by yuviii**

### CODE REVIEW

File: src/app/surveys/surveys.component.ts [hunk 1]
```diff
@@ -56,7 +56,7 @@ export class SurveysComponent implements OnInit, AfterViewInit, OnDestroy {
     private route: ActivatedRoute,
     private stateService: StateService,
     private dialogsLoadingService: DialogsLoadingService,
-    private userService: UserService
+    private userService: UserService,
```
Comment (paulbert) [inline src/app/surveys/surveys.component.ts]: Left in extra comma, not needed.

File: src/app/surveys/surveys.component.ts [hunk 2]
```diff
@@ -75,15 +75,29 @@ export class SurveysComponent implements OnInit, AfterViewInit, OnDestroy {
...
           })
         );
         this.emptyData = !this.surveys.data.length;
+        this.assignCourseIdInsurvey();
         this.dialogsLoadingService.stop();
       });
+  }
+
+  assignCourseIdInsurvey() {
     this.couchService.checkAuthorization(this.dbName).subscribe((isAuthorized) => this.isAuthorized = isAuthorized);
```
Comment (paulbert) [inline src/app/surveys/surveys.component.ts]: This line should remain in `ngOnInit`.  It is unrelated to getting courses.

**Comment by lmmrssa** [comment]
@yuviii could you not add course title on table and just add an icon or button to click which they can go to course.

**Comment by yuviii** [comment]
> @yuviii could you not add course title on table and just add an icon or button to click which they can go to course.

@lmmrssa I had the button previously but again added the course title as per @paulbert comment.

**Comment by lmmrssa** [comment]
> > @yuviii could you not add course title on table and just add an icon or button to click which they can go to course.
> 
> @lmmrssa I had the button previously but again added the course title as per @paulbert comment.

In that case title of course was supposed to be truncated just to show few words. Otherwise it will consume lot of space and other columns will have to shrink.

**Comment by yuviii** [comment]
@lmmrssa @paulbert  Please review the update. The titles are truncated in case of long

**Pull request merged**